generator/src/main.rs

```diff
@@ -114,7 +114,7 @@ async fn main() -> io::Result<()> {
             title: "Contributing",
             links: vec![
                 NavLink {
-                    title: "Overview",
+                    title: "Overview (coming soon)",
```

Comment:
I did this as a separate commit in case you didn't want it. If you don't, just let me know and I'll just drop this commit

Reply:
works for me, the link doesn't work so coming soon is appropriate